apps/ui/src/views/Create.vue

```diff
@@ -1,314 +1,36 @@
 <script setup lang="ts">
-import { clone, getSalt } from '@/helpers/utils';
-import { enabledReadWriteNetworks, getNetwork } from '@/networks';
-import { StrategyConfig } from '@/networks/types';
-import { NetworkID, SpaceMetadata, SpaceSettings } from '@/types';
-
-const PAGES = [
-  {
-    id: 'profile',
-    title: 'Profile'
-  },
-  {
-    id: 'network',
-    title: 'Network'
-  },
-  {
-    id: 'strategies',
-    title: 'Strategies'
-  },
-  {
-    id: 'validations',
-    title: 'Proposal validation'
-  },
-  {
-    id: 'executions',
-    title: 'Executions'
-  },
-  {
-    id: 'auths',
-    title: 'Auths'
+import SpaceCreateOffchain from '@/components/SpaceCreateOffchain.vue';
+import SpaceCreateOnchain from '@/components/SpaceCreateOnchain.vue';
```

Comment:
I would even rename these ideally
```suggestion
import SpaceCreateSnapshot from '@/components/SpaceCreateSnapshot.vue';
import SpaceCreateSnapshotX from '@/components/SpaceCreateSnapshotX.vue';
```